lib/acts_as_flaggable/flag.rb

```diff
@@ -199,4 +199,11 @@ def deletable_by?( user )
     false
   end
 
+  def resolvable_by?( user )
+    if user.is_curator?
+      return true unless user.id === self.flaggable_user_id and flaggable_type != "Taxon"
```

Comment:
Please use `&&` instead of `and`. I don't it doesn't make a difference here and in general we try to wrap assignments in parens which kind of skirts the difference, but it's helpful for consistency... and because every time I see `and` I have to spend a few seconds wondering whether it's going to cause an order of operations bug.